data/meson.build

```
@@ -0,0 +1,39 @@
+#Install icons
+icon_sizes = ['16', '32', '48', '64', '128']
+
+foreach i : icon_sizes
+    install_data(
+        join_paths('icons', i, meson.project_name() + '.svg'),
```

For some reason, I don't remember exactly, I needed to add the png icons as well.
I think the Polkit was not showing the svg icon.

Can you please add the png icons here to be save.

---

I'll take a look on this

---

I fixed this

---

You can even merge the two `install_data` calls.

```suggestion
        join_paths('icons', i, meson.project_name() + '.svg'),
        join_paths('icons', i, meson.project_name() + '.png'),
```